Views/Layout-EmailTemplate.cshtml

```diff
@@ -0,0 +1,45 @@
+@{
+    const string blockName = "emailTemplate";
+}
+
+<!DOCTYPE html>
+<html lang="@Orchard.CultureName()">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width" initial-scale="1">
+    <!--[if !mso]>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <![endif]-->
+    <title>
+        <shape type="EmailTemplate_Title"></shape>
+    </title>
+
+    @await RenderSectionAsync("Head", false)
+
+    <shape type="EmailTemplate_LayoutInjections" prop-layoutModel="@Model"></shape>
+</head>
+<body>
+
+<span class="preheader" style="display: none; max-height: 0px; overflow: hidden;">

```

Comment:
Would the [`hidden` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) work instead of this inline styling?

Reply:
Hidden attribute won't work in Microsoft Outlook as they are using an old IE engine.

Reply:
That confused me becuase "hidden" works fine on IE. Looks like actually Outlook uses Microsoft Word instead of IE (yes, really) since 2007. I found nothing about "hidden" not working specifically, but I will take your word for it.

Reply:
Well, I'm not even close to being an email HTML master, and I don't want to be one either, so don't take my word for it. By the way, I don't see IE here: https://www.w3schools.com/tags/att_hidden.asp.

Reply:
Brah, don't use w3schools. It's not authoritative or even well maintained. People will think you are some sort of amateur. 😜 MDN is the way to go, it's also shorter to type into google.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden#browser_compatibility

Reply:
oops, :D I've just got really close to revealing my ugly secret!